proto/src/determined/log/v1/log.proto

```diff
@@ -0,0 +1,19 @@
+syntax = "proto3";
+
+package determined.log.v1;
+option go_package = "github.com/determined-ai/determined/proto/pkg/logv1";
+
+import "protoc-gen-swagger/options/annotations.proto";
+
+// LogEntry is a log event.
+message LogEntry {
+    option (grpc.gateway.protoc_gen_swagger.options.openapiv2_schema) = {
+        json_schema: {
+            required: ["id", "message"]
```

Comment:
doesn't requiring `message` break the detail about having some non-message logs?

Reply:
this is just annotation but still not marking it is better